src/core/packages/chrome/navigation/types.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { IconType } from '@elastic/eui';
+
+export interface SecondaryMenuItem {
+  id: string;
+  label: string;
+  href: string;
+  iconType?: IconType;
+  external?: boolean;
+}
+
+export interface SecondaryMenuSection {
+  id: string;
+  label: string | null;
+  items: SecondaryMenuItem[];
+}
+
+export interface MenuItem {
+  id: string;
+  label: string;
+  href: string;
```

Comment:
is `href` really required here and this can't be just a sub-menu opener? sounds good if this was a requirement